blynklib_cp.py

```diff
@@ -196,7 +198,20 @@ def _get_socket(self):
             self._state = self.CONNECTING
             self._socket = socket.socket()
             self._socket.connect(socket.getaddrinfo(self.server, self.port)[0][4])
-            self._socket.settimeout(self.SOCK_TIMEOUT)
+            if (self.ssl_cert is not None):
+                self.log('Using SSL socket...')
+                import ssl, os
+                sslContext = ssl.create_default_context()
```

Comment:
Here we can defile create_default_context(cafile=self.ssl_cert)  - so there will be no need to use 
sslContext.load_verify_locations call later